deploy/olm-catalog/0.3.0/submarineraddon.open-cluster-management.io_submarinerconfigs.yaml

```
@@ -131,8 +131,8 @@ spec:
                 clusterName:
                   description: ClusterName represents the name of the managed cluster.
                   type: string
-                infraId:
-                  description: InfraId represents the infrastructure id of the managed cluster.
+                infraID:
+                  description: InfraID represents the infrastructure ID of the managed cluster.
```

We can't change the YAML representation.

```suggestion
                infraId:
                  description: InfraId represents the infrastructure ID of the managed cluster.
```